src/Simplex/Chat/Controller.hs

```diff
@@ -294,11 +294,15 @@ data ChatCommand
   | ExecChatStoreSQL Text
   | ExecAgentStoreSQL Text
   | SlowSQLQueries
+  | APIGetChatTags
```

Comment:
this needs UserId

Reply:
or can just add list of tags to User and load with active user maybe